react/MuiCozyTheme/Menus/index.js

```diff
@@ -1,3 +1,122 @@
-import Menu from '@material-ui/core/Menu'
+import React, { Component } from 'react'
 
-export default Menu
+import MenuButton from '../../Button'
+import PropTypes from 'prop-types'
+
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import Grow from '@material-ui/core/Grow'
+import Popper from '@material-ui/core/Popper'
+import Paper from '@material-ui/core/Paper'
+import MenuList from '@material-ui/core/MenuList'
+import { withStyles } from '@material-ui/core/styles'
+
+const styles = {
+  paper: {
+    '& hr': {
+      margin: '8px 0',
+      border: 0,
+      borderTop: '1px solid var(--silver)'
+    }
+  }
+}
+
+class Menu extends Component {
+  state = { open: false, anchorEl: null }
+
+  toggle = event => {
+    const { currentTarget } = event
+    this.setState(state => ({
+      anchorEl: currentTarget,
+      open: !state.open
+    }))
+  }
+
+  close = () => {
+    this.setState({
+      open: false
+    })
+  }
+
+  render() {
+    const {
+      label,
+      disabled,
+      buttonClassName,
+      component,
+      children,
+      placement,
+      classes
```

Comment:
`s/classes/className` ?

Reply:
Mmmmh nope, `classes` refers to classes set in the `styles` const. 
Something M-UI proposes to override classes.